src/fsharp/FSComp.txt

```diff
@@ -1459,8 +1461,10 @@ keywordDescriptionUntypedQuotation,"Delimits a untyped code quotation."
 3216,itemNotFoundInTypeDuringDynamicCodeGen,"%s '%s' not found in type '%s' from assembly '%s'. A possible cause may be a version incompatibility. You may need to explicitly reference the correct version of this assembly to allow all referenced components to use the correct version."
 descriptionWordIs,"is"
 notAFunction,"This value is not a function and cannot be applied."
-notAFunctionButMaybeIndexerWithName,"This value is not a function and cannot be applied. Did you intend to access the indexer via %s.[index] instead?"
-notAFunctionButMaybeIndexer,"This expression is not a function and cannot be applied. Did you intend to access the indexer via expr.[index] instead?"
+notAFunctionButMaybeIndexerWithName,"This value is not a function and cannot be applied. Did you intend to access the indexer via '%s.[index]' instead?"
```

Comment:
instead is redundent:
"This value is not a function and cannot be applied. Did you intend to access the indexer via '%s.[index]'?"

Is just as good and less colloquial.

The next few messages have the same issue.

Reply:
Thanks!